Source/Scene/ImplicitSubtree.js

```diff
@@ -393,6 +473,32 @@ function requestActiveBuffers(subtree, bufferHeaders, internalBuffer) {
   });
 }
 
+function requestExternalBuffer(bufferHeader, baseResource) {
+  var bufferResource = baseResource.getDerivedResource({
+    url: bufferHeader.uri,
+  });
+  var cacheKey = ResourceCacheKey.getExternalBufferCacheKey({
+    resource: bufferResource,
+  });
+
+  var cacheResource = ResourceCache.get(cacheKey);
+  if (!defined(cacheResource)) {
+    cacheResource = new BufferCacheResource({
+      cacheKey: cacheKey,
+      resource: bufferResource,
+    });
+
+    ResourceCache.load({
```

Comment:
Any loaded buffers will need to be explicitly unloaded.  `ImplicitSubtree` may need a destroy function and `Cesium3DTileset` may need to call each subtree's destroy function in its destroy function.

Reply:
Or we decided that caching subtree buffers isn't that important and we go back to the old way.

Reply:
How should we keep track of when these need to be destroyed? each tile has a reference to the subtree, but the tileset does not.

Reply:
I talked with @lilleyse, one idea is to only destroy subtrees from the placeholder tiles, and treat all the transcoded tiles' references as weak references. I'll look into this tomorrow.

Reply:
I really like this new approach. Much simpler than what I suggested.